_data/pricing.yml

```diff
@@ -1,26 +1,44 @@
-tier1:
-  title: Up to 5 users
-  users_max: 5
-  price: [500, 795]
-  checkout:
-    - price: 500
-      url: /checkout/?plan=1
-    - price: 795
-      url: /checkout/?plan=1&addon=dedicated-support
-  details: |
-    monthly flat fee
+subscription_only:
+  tier1:
+    title: Up to 5 users
+    users_max: 5
+    price: 795
+    checkout:
+      - price: 795
+        url: /checkout/?plan=1
+    details: |
+      monthly flat fee
 
-tier2:
-  title: Users 6 - 50
-  users_max: 50
-  price: [79, 99]
-  checkout:
+  tier2:
+    title: Users 6 - 50
+    users_max: 50
+    price: 79
+    checkout:
+        - price: 79
+          url: /checkout/?plan=2
+    details: |
+      per <span class="help-text light" title="A user is any human or machine that makes use of any of Gruntwork's services (Infrastructure as Code Library, DevOps Training Library, Dedicated Support)" data-toggle="tooltip">user</span> per month
+
+dedicated_support:
+  tier1:
+    title: Up to 10 users
```

Comment:
Why 10 here but 5 there? It makes the price points hard to compare...

Reply:
10 users for $2,745/month just didn't seem like a good enough value to me. Do you think we should make the base library $795 for 10 users to keep them the same?

Reply:
You mean 5 users for $2,745?

Reply:
Sorry, yes, I mant 5 users for $2,745.

Reply:
Alright, sure. I have no compelling evidence one way or the other.

Setting both to 10 users makes sense then.

However, sanity check: will this cause any unexpected behavior amongst existing customers? Will those paying for, say, 9 users, ask to have their price reduced?

Reply:
Hmm, possibly. This new, interim pricing is less expensive in terms of the number of users, but much more expensive for Dedicated Support. And almost all the customers with a large number of users also have dedicated support, which would mean that their current pricing is still probably more favorable for them.

That being said, if a customer is moved to the point of saying something about the new pricing, I'm inclined to allow them to switch to it in most cases. Any lingering concerns with that?

Reply:
just spotted this in my timeline, 
I would always offer the new pricing to a current customers, if it's cheaper, then it should be on the basis of a "new contract" so 12-month commitment if they wish to fallback, they go back to old arrangement with a 12-month commitment on the "old pricing" 
either way, a change of subscription should mean 12 months commitment, 
Many mobile providers do it that way, as to break even on the dialog required they need a contract extension

Reply:
@grealish That sounds reasonable: Any customer that wants to adopt the new pricing can at any time, and starts a new contract at that time.